cmd/minishift/cmd/util/util.go

```diff
@@ -71,3 +72,17 @@ func UnpackAddons(addonsDir string) error {
 
 	return nil
 }
+
+func ShowProgressDots(progressDots chan bool) {
```

Comment:
+1

Reply:
I think we should name it `StartProgressDots`.

Reply:
Also I think we should move this function to minishift pkg/Util.